### PR TITLE
feat : added alert and button component

### PR DESCRIPTION
**Pull request by someshtiwarii**
Added alert and button component

### CODE REVIEW

**Review by suryxks — APPROVED** [review]
Everything looks good
Semantic html tags can be used instead of divs .<nav> can be used for the sidebar and <section> tag can be used instead of giving <div class="section">

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+}
+
+.alert {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5rem 1rem 0.5rem 1rem;
```
Comment (gopalbbharadva) [inline components/alert/alert.css:17]: Alternative: "padding 0.5rem 1rem". it wil affect the same.

**Comment by BelideNikhil** [comment]
good to go!

**Review by gopalbbharadva — APPROVED** [review]

File: components/alert/alert.css (hunk 1 above)
Comment (someshtiwarii) [reply components/alert/alert.css:17]: okay Gopal i will do that .. Thankyou

**Pull request merged**